proto/kvrpcpb.proto

```diff
@@ -446,6 +446,7 @@ message RawPutRequest {
     bytes value = 3;
     string cf = 4;
     uint64 ttl = 5;
+    bool for_cas = 6;
```

Comment:
How about `check_cas`?

Reply:
I think there is no difference between them....

Reply:
`for_cas` seems the key is put for CAS purpose. However, the field is meant to be participated in CAS checking.